che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/Messages.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - API 
+ *   SAP           - initial implementation
+ *******************************************************************************/
+package org.eclipse.che.git.impl.jgit;
+
+public class Messages {
```

Comment:
Do we need this class only for storing error messages in one place? If yes maybe it would be better to use constant class or hardcode values of messages near usage for that and remove Messages and ExternalStringsSource classes.
I understand motivation to use this classes but anyway for now we don't have such feature for another APIs and we should do it in the same way.
WDYT?